legal-api/report-templates/certificateOfIncorporation.html

```diff
@@ -6,7 +6,7 @@
     <title>Certificate of Incorporation</title>
     <meta charset="UTF-8">
     <meta name="author" content="BC Registries and Online Services">
-    [[certificate-of-incorporation/style.html]]
+    [[common/certificateStyle.html]]
```

Comment:
Can you please attach a test PDF of this report as well (since it was modified, to verify that it's OK)? Thanks!

Reply:
[certificateOfIncorporation3.pdf](https://github.com/bcgov/lear/files/6030942/certificateOfIncorporation3.pdf)

I also added a test for this one.